studio/components/interfaces/Organization/NewOrg/NewOrgForm.tsx

```diff
@@ -305,10 +305,11 @@ const NewOrgForm = ({ onPaymentMethodReset }: NewOrgFormProps) => {
                     readOnly
                     className="w-64"
                     size="small"
-                    value={`•••• •••• •••• ${paymentMethod.card.last4}`}
+                    value={`•••• •••• •••• ${paymentMethod.card?.last4}`}
```

Comment:
nit: otherwise it might look a little weird
```suggestion
                    value={`•••• •••• •••• ${paymentMethod.card?.last4 ?? '••••'}`}
```